docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -943,6 +945,37 @@ xrdsst client delete --ss <SECURITY_SERVER_NAME> --client <CLIENT_ID>
 
 The members or subsystem must be unregistered from the security server in order to delete it.
 
+##### 4.2.6.7 Client list
+
+* Access rights: Any role
+
+List clients (subsystem and members) can be done with:
+```
+xrdsst client list --ss <SECURITY_SERVER_NAME>
+```
+* <SECURITY_SERVER_NAME> name of the security server, e.g. ss1
+
+╒══════════════════╤════════════╤════════════════╤═══════════════╤═══════════════╤═════════════╤═════════╤════════════╤═════════════════╕
+│ ID               │ INSTANCE   │ MEMBER CLASS   │   MEMBER CODE │ MEMBER NAME   │ SUBSYSTEM   │ OWNER   │ STATUS     │ HAS SIGN CERT   │
+╞══════════════════╪════════════╪════════════════╪═══════════════╪═══════════════╪═════════════╪═════════╪════════════╪═════════════════╡
+│ DEV:COM:12345    │ DEV        │ COM            │         12345 │ COMPANY       │             │ False   │ SAVED      │ True            │
+├──────────────────┼────────────┼────────────────┼───────────────┼───────────────┼─────────────┼─────────┼────────────┼─────────────────┤
+│ DEV:ORG:111      │ DEV        │ ORG            │           111 │ NIIS          │             │ True    │ REGISTERED │ True            │
+├──────────────────┼────────────┼────────────────┼───────────────┼───────────────┼─────────────┼─────────┼────────────┼─────────────────┤
+│ DEV:ORG:111:TEST │ DEV        │ ORG            │           111 │ NIIS          │ TEST        │ False   │ REGISTERED │ True            │
+╘══════════════════╧════════════╧════════════════╧═══════════════╧═══════════════╧═════════════╧═════════╧════════════╧═════════════════╛
+
+* ID client id
+* INSTANCE client instance
+* MEMBER CLASS client member class
+* MEMBER CODE client member code
+* MEMBER NAME client member name
+* SUBSYSTEM client subsystem code (empty for members)
+* OWNER true if the client is the owner of the security server
+* STATUS client status between SAVED, REGISTRATION IN PROGRESS, REGISTERED, GLOBAL ERROR, DELETION IN PROGRESS, DELETED
+* HAS SIGN CERT true if the client has a valid sign certificate
+
+
 ##### 4.2.6.6 Client change owner
```

Comment:
Client change owner has the same paragraph numbering as Client delete, both are 4.2.6.6